Task 3/system/Coding/src/app/PositionControl.h

```diff
@@ -25,30 +25,26 @@
 
 /* PROTOTYPES *************************************************************************************/
 /** 
- * @brief starts timer1 to measure the Time in SearchTrackState
+ * @brief updates the sensorvalues, must be called before any of the other functions are beein called
 */
-extern void SearchTrackState_enterStartTimer1(void);
+extern void PositionControl_UpdateSensorValues(void);
 
 /** 
- * @brief Allgorithm to search the TrackLine, after it has been lost 
+ * @brief simple PID controlling allgorithm to follow the trackline
 */
-extern void SearchTrackState_processFindTrackLine(void);
 
-/**
- * @brief stops timer1
-*/
-extern void SearchTrackState_exitStopTimer1(void);
+extern void PositionControl_DriveOnTrack(void);
```

Comment:
Function name has to start with lowerCase letter.
```suggestion
extern void PositionControl_driveOnTrack(void);
```